### PR TITLE
Empty location

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/abstract-leveldown/issues/59

### CODE REVIEW

File: abstract-leveldown.js [hunk 1]
```diff
@@ -11,6 +11,9 @@ function AbstractLevelDOWN (location) {
   if (typeof location != 'string')
     throw new Error('constructor requires a location string argument')
```
Comment (juliangruber) [inline abstract-leveldown.js]: i'd merge the two into something like

```js
if (typeof location != 'string' || !location.length)
  throw new Error('constructor requires a non empty location string')
```

that should be enough information

**Comment by juliangruber** [comment]
otherwise lgtm :)

File: abstract-leveldown.js (hunk 1 above)
Comment (ralphtheninja) [inline abstract-leveldown.js]: Good call. I'll update.

**Comment by ralphtheninja** [comment]
```
$ npm publish
+ abstract-leveldown@2.2.1
```

**Pull request merged**